### DISCUSSION TITLE
import fulltos from tosback

### DISCUSSION THREAD

**Discussion by michielbdejong** [tosdr/CrispCMS#57] [General]
https://github.com/tosdr/tosdr.org/wiki/Specification:-services#fulltos needs to be deprecated.

Compare https://github.com/tosdr/tosback2/blob/master/rules/google.com.xml
* Privacy Policy http://www.google.com/intl/en/policies/privacy/ //div[@class='maia-article'] reviewed
* Terms of Service http://www.google.com/intl/en/policies/terms/" //div[@class='maia-article'] reviewed
* Google Analytics Terms of Service https://www.google.com/analytics/terms/us.html //div[@class='main']

To https://github.com/tosdr/tosdr-build/blob/master/src/services/google.json

There is an undocumented obj.tosback2 field which is used in https://github.com/tosdr/tosdr-build/blob/master/build/buildIndexes.js#L97-L104

can we change the buildIndexes script so that it uses the data from the tosback2 repo? maybe add that as a submodule?

**Comment by michielbdejong** [comment]
First, I'll work on https://github.com/tosdr/phoenix/issues/225, and there I'll ignore both `obj.fulltos` and `obj.tosback2`.

To link tosdr services to tosback rules, we can say the service's `obj.url` can be the tosback rule filename minus the '.xml'

**Comment by JustinBack** [comment]
Is this still relevant? Since we are switching to a different crawler now
